File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.cs
using System;

public class Solution {
    public int solution(string ineq, string eq, int n, int m) {
        int answer = 0;
        if(ineq[0] == '>' && eq[0] == '=')
        {
            if(n >= m)
                return 1;
            else
                return 0;
        }
        else if(ineq[0] == '<' && eq[0] == '=')
        {
            if(n <= m)
                return 1;
            else
                return 0;
        }
        else if(ineq[0] == '<' && eq[0] == '!')
        {
            if(n < m)
                return 1;
            else
                return 0;                
        }
        else
        {
            if(n > m)
                return 1;
            else 
                return 0;
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string ineq, string eq, int n, int m) {
    int answer = 0;
    string s = ineq + eq;
    if(s == ">=") answer = n >= m ? 1 : 0;
    if(s == "<=") answer = n <= m ? 1 : 0;
    if(s == ">!") answer = n > m ? 1 : 0;
    if(s == "<!") answer = n < m ? 1 : 0;
    return answer;
}